### DISCUSSION TITLE
Edit element values through javascript

### DISCUSSION THREAD

**Discussion by lakii22** [frangoteam/FUXA#562] [Q&A]
Hi, I would like to know how I can access the elements from my views through JS.

Example: Lets say that I've a "Text Output" element and I want to set the content through JS without using a variable. So everytime that I execute "script" the value changes.

I know that as we are in the server side I can't use "Document" for selection, so Document.querySelector or Document.getElementById aren't an option to reach the elements throught scripts.

Is there a way to achieve this ?

Thanks.

**Comment by unocelli** [comment]
Hi, I don't understand why you don't want to use a variable to communicate with the client via script? there aren't other options.

**Reply by lakii22** [reply]
Hi @unocelli, I was trying to reduce the amount of server variables that I was using before for a performance improvement and easy maintenance of the variables... As my project could get a lot of similar machines that uses the same kind of sensors, I was trying to make some changes directly on the JS to use less variables from FUXA server. 

After some thought I could reduce the amout of variables with a simple switch case as follow:
![image](https://user-images.githubusercontent.com/7461434/198099814-5de29273-889e-4cd7-8f1f-a197d3a6fbf2.png)

So I can use the same view to cicle through a lot of equipments without problems. 

I know that this is a simple thing but my goal is to make the setup of a lot of machines/devices quickier.
